README.md

```diff
@@ -29,12 +29,32 @@ Currently, @redocly/openapi-cli supports these features:
 
 Unlike other OpenAPI linters, `@redocly/openapi-cli` defines the possible type tree of a valid OpenAPI definition and then traverses it. This approach is very similar to how linters for programming languages work and results in major performance benefits over other approaches. Extend functionality at different points in the lifecycle with preprocessors, rules, and decorators.
 
-## TLDR
+## Usage
 
-`npx @redocly/openapi-cli lint path-to-root-file.yaml`
+### Node
 
-## [Read the docs](https://redoc.ly/docs/cli/)
+```
+npx @redocly/openapi-cli lint path-to-root-file.yaml
+```
+
+### Docker
+
+To give the docker container access to the OpenAPI spec files, you need to
+mount the containing directory as a volume. Assuming the spec is rooted
+in the current working directory, you need the following command:
+
+```
+docker run --rm -v $PWD:/spec redocly/openapi-cli lint path-to-root-file.yaml
+```
 
+To build and run with a local image, run the following from the project root:
+
+```
+docker build -t openapi-cli .
+docker run --rm -v $PWD:/spec oepnapi-cli lint path-to-root-file.yaml
+```
+
+## [Read the docs](https://redoc.ly/docs/cli/)
```

Comment:
Was the removal of the blank line below this one an intentional change?

Reply:
Done.